src/main/webapp/payload.ftl

```diff
@@ -313,8 +381,177 @@
 	</div>
   </body>
 
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+<script src="https://cdn.datatables.net/1.10.20/js/jquery.dataTables.min.js"></script>
 <script type="text/javascript">
-	$(function() {
+    var expRows = {
+        <#list rows as row>
+        <#assign data = row.data>
+        <#if data.associatedPullRequest?has_content || data.associatedUnrelatedPullRequest?has_content || data.dependsOn?has_content>
+        "${data.payloadDependency.label}": {
+            <#if data.associatedPullRequest?has_content || data.associatedUnrelatedPullRequest?has_content>
```

Comment:
Thank you. Does it need to add `data.dependsOn?has_content` as well? I understand that the "Expand All" is disabled with current change. But the simple "expand" for each row only works for associated pull request, doesn't work for depends on issues.

Reply:
We currently diplay both:
![Screenshot from 2020-01-20 12-33-26](https://user-images.githubusercontent.com/1999074/72724008-7270be00-3b82-11ea-8790-7760bd417692.png)
There shouldn't be a structural change in the FreeMarker code, I just moved some parts around (and added a macro).